### PR TITLE
Update copy of help screen

### PR DESCRIPTION
**Pull request by irfano**
This updates the copy of the help screen according to the design.
(_Internal ref:_ qk5s4q4mZsXknwhMWo2vIY-fi-3882%3A14189)

To reviewer: It'd be easier if you follow commits to avoid the distraction from the last commit ([Remove unused string resources](https://github.com/wordpress-mobile/WordPress-Android/commit/5244dc304a9a9184706eadf4132def2018c9c1aa)).

||before|after|
|-|-|-|
|WP Me|||
|JP Me|||
|WP Help|||
|JP Help|||

To test:
**Community forums state**
1. Launch the WP app. 
2. Log in with an account that doesn't have a paid plan.
3. Check the design file. Compare texts with the app's help screen.

**Contact support state**
1. Launch the JP app and log in. 
2. Check the design file. Compare texts with the app's help screen.

## Regression Notes
1. Potential unintended areas of impact
Some changes may not look right in the context.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Nothing.

3. What automated tests I added (or what prevented me from doing so)
Automated tests don't include text changes.

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by ravishanker — COMMENTED** [review]
@irfano changes need to be made only in the strings.xml.  The rest of the language files get updated through fastlane and transaltion before release.

One way to make it is easy is to delete any old strings and create new ones to ensure they get deleted in the translated files, and news ones get translated.

For e.g. en-rAU/strings.xml still has "Help and Support" for help_screen_title.

**Comment by irfano** [comment]
> @irfano changes need to be made only in the strings.xml. The rest of the language files get updated through fastlane and transaltion before release.
>
> One way to make it is easy is to delete any old strings and create new ones to ensure they get deleted in the translated files, and news ones get translated.

Are unused string resources removed during the translation process? 
I can't delete any strings only from the default `strings.xml`, then I would get a lint error about missing translation.

I can revert the [last commit](https://github.com/wordpress-mobile/WordPress-Android/pull/18002/commits/5244dc304a9a9184706eadf4132def2018c9c1aa), but I want to be sure about my concern.

> For e.g. en-rAU/strings.xml still has "Help and Support" for help_screen_title.

Yes, I updated `help_screen_title` in the default `strings.xml` and I expected other translations will be updated during the process since `help_screen_title`'s value is updated on the default `strings.xml`. Isn't it correct?

**Comment by ravishanker** [comment]
> Are unused string resources removed during the translation process? I can't delete any strings only from the default `strings.xml`, then I would get a lint error about missing translation.
> 
Yes, it is better to create a new key with the new string and use it in the layout and delete the unused old string.  It would guarantee ui shows the new string during development and also gets translated through polyglot.

Here's a recent relevant discussion  p1670886485721879-slack-CC7L49W13

**Comment by irfano** [comment]
Thank you for sharing the discussion, @ravishanker. I understand the reason for your request. I have made the necessary changes. Could you please review it again?

File: WordPress/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -31,6 +31,7 @@
     <!-- Screen titles -->
     <string name="release_notes_screen_title">Release notes</string>
     <string name="help_screen_title">Help and Support</string>
```
Comment (ravishanker) [inline WordPress/src/main/res/values/strings.xml]: This is not in use anymore right?  Then it should be removed.

File: WordPress/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -2247,7 +2248,9 @@
     <string name="wordpress_help_center_button">WordPress help center</string>
     <string name="browse_our_faq_button">Browse our FAQ</string>
     <string name="my_tickets">My Tickets</string>
```
Comment (ravishanker) [inline WordPress/src/main/res/values/strings.xml]: Same here, My Tickets should go

File: WordPress/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -2385,6 +2388,7 @@
     <string name="me_profile_photo">Profile Photo</string>
     <string name="me_btn_app_settings">App Settings</string>
     <string name="me_btn_support">Help &amp; Support</string>
```
Comment (ravishanker) [inline WordPress/src/main/res/values/strings.xml]: and this me_btn_support

**Review by ravishanker — APPROVED** [review]
LGTM 👍 

Please remove the unused strings before merging

File: WordPress/src/main/res/values/strings.xml (hunk 3 above)
Comment (irfano) [reply WordPress/src/main/res/values/strings.xml]: Removed: e1f1d73a54abf855e7f4de45414f98916a64d4b5

File: WordPress/src/main/res/values/strings.xml (hunk 2 above)
Comment (irfano) [reply WordPress/src/main/res/values/strings.xml]: Removed: e1f1d73a54abf855e7f4de45414f98916a64d4b5

File: WordPress/src/main/res/values/strings.xml (hunk 1 above)
Comment (irfano) [reply WordPress/src/main/res/values/strings.xml]: Removed: e1f1d73a54abf855e7f4de45414f98916a64d4b5

**Pull request merged**